src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc3-24126-00"
```

Comment:
Thanks for the heads up. When that happens, this version will automatically be updated by `update-dependencies.ps1`.